llama-index-integrations/node_parser/relational/llama-index-node-parser-relational-dashscope/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.node_parser.relational.dashscope"
+
+[tool.llamahub.class_authors]
+DashScopeEmbedding = "phantomgrapes"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Ruixue Ding <ada.drx@alibaba-inc.com>"]
+description = "llama-index node_parser relational dashscope integration"
+license = "MIT"
+name = "llama-index-node-parser-relational-dashscope"
```

Comment:
Since this is the first node-parser integration, we probably only need to put it as `llama-index-node-parser-dashscope`, as well as modifying the folder structure to be `from llama_index.node_parser.dashscope import DashScopeJsonNodeParser`